docs/nima/guides/webclient.adoc

```diff
@@ -0,0 +1,411 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Helidon Nima WebClient Guide
+:description: Helidon WebClient
+:keywords: helidon, WebClient, web, client
+:rootdir: {docdir}/../..
+
+include::{rootdir}/includes/nima.adoc[]
+
+This guide describes how to create a sample Helidon SE project
```

Comment:
For completeness (and monotony), check for SE uses in text and example code.

Also, the Níma WebClient is not (or at least we won't highlight that it can be) reactive. It's synchronous. This page will need some work to change that emphasis.